### ISSUE TITLE
New update blocking the backlight from going on

### ISSUE THREAD

**Issue by TheDarkOn3** [DarkFlippers/unleashed-firmware#563]
This issue is noticed with the 59r.tng file. Was the code accidentally removed from the new update?

**Comment by xMasterX** [comment]
User related issues like this one, can not be fixed, I don't know who could fix inability to read
Do you understand that other color LEDs will not grow up magically in your flipper? 
How many times I need to add this (only for hardware modded flippers) so it won't be ignored again and again?

**Comment by TheDarkOn3** [comment]
I am saying it’s no lighting up whatsoever but still have the image. It’s something in the code that causes this. I am not saying anything about the color mod. Sent from my iPhoneOn Jul 28, 2023, at 19:00, MMX ***@***.***> wrote:﻿
User related issues like this one, can not be fixed, I don't know who could fix inability to read
Do you understand that other color LEDs will not grow up magically in your flipper?
How many times I need to add this (only for hardware modded flippers) so it won't be ignored again and again?

**Comment by SkeletonMan03** [comment]
You didn't read the instructions. The version that has r at the end is for Flipper Zeros that have had their screens modded with an RGB board. You do not have that RGB mod. Therefore, you get no backlight with the r version installed. Install one of the other versions. Actually read what it says above the files on the release page.

**Comment by TheDarkOn3** [comment]
I noticed that myself (I’m kinda noob with some knowledge on programming). I just saw that. To everyone thx for the help. Sent from my iPhoneOn Jul 28, 2023, at 21:27, SkeletonMan03 ***@***.***> wrote:﻿
You didn't read the instructions. The version that has r at the end is for Flipper Zeros that have had their screens modded with an RGB board. You do not have that RGB mod. Therefore, you get no backlight with the r version installed. Install one of the other versions. Actually read what it says above the files on the release page.
